src/components/layout/Undernames/Undernames.tsx

```diff
@@ -292,7 +293,8 @@ function Undernames() {
                 ) : (
                   <></>
                 )}
-                {action === 'create' || action === 'edit' ? (
+                {action === UNDERNAME_TABLE_ACTIONS.CREATE ||
+                action === UNDERNAME_TABLE_ACTIONS.EDIT ? (
```

Comment:
consider a few types to help here:

const createOrUpdateUndernameInteractions = [UNDERNAME_TABLE_ACTIONS.CREATE, UNDERNAME_TABLE_ACTIONS.EDIT] as const;
const destructiveUndernameInteractions =  [UNDERNAME_TABLE_ACTIONS.REMOVE] as const;
const allUndernameInteractions = [
    ...createOrUpdateUndernameInteractions,
    ...destructiveUndernameInteractions
] as const;
```
and check against these arrays via `includes()`

Reply:
added